src/content/getting-started/designers/designers.md

```diff
@@ -3,32 +3,61 @@ label: Getting Started
+The Design Kit is available through GitHub and internally to IBMers through Box Drive. The kti can be used as a normal file in the Sketch application or you can load the file as a Sketch Library source.
 
-### 2. Open the Design Kit in Sketch
-
-#### Why Sketch?
+### Why Sketch?
 
 <a href="https://www.sketchapp.com/" target=blank>Sketch</a> is the industry standard design tool for UI design.
 
 You can purchase a license for $99/year. If you donʼt have a license for Sketch, you may sign up for a 30-day free trial. Sketch is easy to learn, and they offer many [tutorials](https://medium.com/sketch-app-sources/helpful-guide-to-getting-started-with-sketch-ecab4b5738bb) for further learning.
 
-![Carbon Design Kit](images/designers-3.png)
+### Releases
+
+The Carbon Design System team often makes improvements to the Design Kit. We track all release updates in our <a href="https://github.com/carbon-design-system/carbon-design-kit/releases" target=blank>change log</a>, so check back frequently!   
+
+## Download from GitHub
+
+1. Download the Kit
+   - <p>Go to the <a href="https://github.com/carbon-design-system/carbon-design-kit" target=blank> Carbon Design Kit</a> repo and click the folder for latest version of the Carbon Design Kit.
+   - On the next page click `Carbon Design Kit.sketch`
+   - Then click the “View Raw” link to begin downloading the kit.</p>
+![Carbon Design Kit repo screenshot](images/designers-2.png)
+
+2. Open the Carbon Design Kit in Sketch
+   - Open and use the kit like any other Sketch file.
+   - Use the left panel in the file to navigate through the different pages of the kit.
+![Kit file screenshot](images/designers-3.png)
+
+3. Install the Design Kit as a Sketch Library
+   - The same Carbon Design Kit file can also be installed as a Sketch Library.
+   - Follow the [instructions](https://github.com/IBM/carbon-design-kit/wiki/Sketch-Libraries-Overview) on the repo wiki page to set the kit up as Sketch Library.
+![Sketch Library screenshot](images/designers-4.png)  
+
+4. Get the updates
+	- To update to the latest Carbon library version you will have to download the newest version of the Carbon Design Kit each time it is released and replace the previous version.
+	- Follow the specific [instructions](https://github.com/IBM/carbon-design-kit/wiki/Sketch-Libraries-Overview) on the wiki to ensure a smooth transition.
 
-### 3. Using the Design Kit
+## Use with Box Drive
 
-The Design Kit contains core visual styles, components, color swatches, iconography, and templates. It also has a collection of symbols and text styles for rapid design and layout prototyping. Each is designated its own page, which may be accessed via the left side panel.
+*For IBM internal users only.* Open source users should use the download from GitHub method. Using the kit through Box Drive enables automatic updates to your Carbon Design Kit library without having to manually reload down the kit everytime as in the download from GitHub method.
```

Comment:
*For IBM internal users only.* Using Box Drive sync will ensure that your design kit file will always be up to date, eliminating the need to manually download new versions as they are released.